ci/run-tests.sh

```diff
@@ -44,7 +44,8 @@ pushd "$(dirname "$0")"
                 -unattended \
                 -nullRHI \
                 -run=GenerateSnapshot \
-                -MapPaths="${TEST_REPO_MAP}"
+                -MapPaths="${TEST_REPO_MAP}" \
+            || true
```

Comment:
What's this do?

Reply:
make sure the script doesn't fail if it runs into errors. Which seems to happen because of the cook